docs/metrics.md

```diff
@@ -0,0 +1,174 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
+
+* Do people use the containers feature?
+  * Click to create new container tab
+* Do people who use the containers feature continue to use it?
+  * Retention: opening a second container tab (second tab in the same container, or a tab in a second container?)
+* What containers do people use?
+  * Container name (should we record the names of custom container created by users?)
+    * \+ Number of tabs in the container (when should we measure this? on every tab open?)
+* Do people edit their containers? (do we care about editing built-in containers vs. user-created containers?)
+  * Click on "Edit Containers"
+  * Click to edit a single container
+    * Click "OK"
+  * Click to delete a single container
+    * Click "OK"
+  * Click to add a container
+    * Click "OK"
+* How do people create new container tabs?
+  * Click to create new container tab
+    * \+ `entry-point` value: "tab-bar" or "pop-up"
+* Do people sort the tabs?
+  * Click sort
+    * \+ Number of tabs when clicked
+  * average number of container types displayed on sort (what does this mean?)
+* Do users hide container tabs?
+  * Click hide
+    * \+ Number of tabs when clicked
+    * \+ `hiddenTimestamp`
+  * Click show
+    * \+ Number of tabs when clicked
+    * \+ `shownTimestamp`
+* How many containers do users have hidden at the same time? (when should we measure this? each time a container is hidden?)
+* Do users move container tabs to new windows?
+  * Click move
+    * \+ Number of tabs when clicked
+* Do users pin container tabs? (do we have existing Telemetry for pinning?)
+* Do users change URLs in a container tab? (sounds like it could be a flood unless we only record the first URL change?)
```

Comment:
Is this something you really need an answer to?  This seems more about browsing habits and, yeah, would be a flood of info